src/sinks/aws_kinesis_streams/config.rs

```diff
@@ -177,14 +213,11 @@ impl GenerateConfig for KinesisSinkConfig {
 struct KinesisRetryLogic;
 
 impl RetryLogic for KinesisRetryLogic {
-    type Error = RusotoError<PutRecordsError>;
+    type Error = SdkError<PutRecordsError>;
     type Response = KinesisResponse;
 
     fn is_retriable_error(&self, error: &Self::Error) -> bool {
-        match error {
-            RusotoError::Service(PutRecordsError::ProvisionedThroughputExceeded(_)) => true,
```

Comment:
The new `is_retriable_error` function for the new AWS SDK doesn't appear to handle this error. Given it is service specific, I could see keeping a `match` here to catch it and fallback to `is_retriable_error`.